webtau-groovy/src/main/groovy/org/testingisdocumenting/webtau/cfg/WebTauGroovyFileConfigHandler.groovy

```diff
@@ -75,6 +81,19 @@ class WebTauGroovyFileConfigHandler implements WebTauConfigHandler {
         setupDbDataSourceProviders(parsedConfig)
     }
 
+    private static void handlePersonas(WebTauConfig cfg, Map<String, ?> personasConfig) {
+        def personas = personasConfig.get(PERSONAS_KEY)
+        if (!personas) {
+            return
+        }
+
+        personas.each { String personaId, personaConfig ->
+            cfg.acceptConfigValues(SOURCE, personaId, personaConfig)
+        }
+
+        personasConfig.remove(PERSONAS_KEY)
```

Comment:
Why remove?

Reply:
1. in case we move to a proper not string based config,`John { configValue = 'a' }` would not generate config values like `personas.John.configValue`
2. I don't want to encourage access to other Persona's config values crossing boundaries

Reply:
I see, makes sense.